src/Amazon.Extensions.CognitoAuthentication/CognitoAuthenticationClasses.cs

```diff
@@ -111,19 +111,35 @@ public class InitiateRefreshTokenAuthRequest
     }
 
     /// <summary>
-    /// Class containing the necessary properities to respond to an SMS MFA authentication challenge
+    /// Class containing the necessary properities to respond to an MFA authentication challenge
     /// </summary>
-    public class RespondToSmsMfaRequest
+    public class RespondToMfaRequest
     {
         /// <summary>
         /// The session ID for the current authentication flow.
         /// </summary>
-        public string SessionID { get; set; }
+        public virtual string SessionID { get; set; }
```

Comment:
Why was SessionID changed to be virtual?

Reply:
This is a new class, and RespondToSmsMfaRequest inherits from it. This was added as a virtual so inheriting classes could override the properties or methods on this new base class